src/lib/LBT/LBT.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "POWERMGNT.h"
+#include "LQCALC.h"
+#include "SX1280Driver.h"
+
+extern LQCALC<100> LBTSuccessCalc;
+extern volatile bool LBTFakeTXdoneISR;
```

Comment:
Not really liking the name of this flag. Perhaps just remove the ISR part as it's not an ISR.

Reply:
agree, terrible name... how about just LBTChannelBusy?

Reply:
Yes, thats a lot more descriptive